buildah/tests/resources.yaml

```diff
@@ -1,22 +1,21 @@
 ---
 apiVersion: tekton.dev/v1alpha1
 kind: PipelineResource
-metadata:
-  name: nocode
-spec:
-  type: git
-  params:
-    - name: revision
-      value: master
-    - name: url
-      value: https://github.com/kelseyhightower/nocode
----
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
 metadata:
   name: image
 spec:
   type: image
   params:
     - name: url
       value: localhost:5000/nocode
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: buildah-source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
```

Comment:
is it working everywhere? what about CI that doesn't have PV available ? (ie: our downstream CI :D)

Reply:
We are already doing that for `buildpacks` :thinking: So my guess is yes, otherwise, we will need to think of something…

Reply:
Bah i let you fix downstreams ci then 😝